com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/configuration/service/ObjectStoreServiceInfoCreator.java

```diff
@@ -0,0 +1,44 @@
+package com.sap.cloud.lm.sl.cf.web.configuration.service;
+
+import java.util.Map;
+
+import org.springframework.cloud.cloudfoundry.CloudFoundryServiceInfoCreator;
+import org.springframework.cloud.cloudfoundry.Tags;
+
+public class ObjectStoreServiceInfoCreator extends CloudFoundryServiceInfoCreator<ObjectStoreServiceInfo> {
+
+    private static final String OBJECT_STORE_LABEL = "objectstore";
+    private static final String OBJECT_STORE_AWS_PLAN = "s3-standard";
+    private static final String OBJECT_STORE_INSTANCE_NAME = "deploy-service-os";
+
+    public ObjectStoreServiceInfoCreator() {
+        super(new Tags(OBJECT_STORE_LABEL), "");
+    }
+
+    @Override
+    public boolean accept(Map<String, Object> serviceData) {
```

Comment:
If you put OBJECT_STORE_AWS_PLAN in the Tags, maybe you will not need the method at all.

Reply:
The super.accept() does not verify plan, but only label or tags. I want to only consume the service if it is running with aws-s3 as backing service and that is reflected by the plan.